docs/reviewing-release-prs.md

```diff
@@ -0,0 +1,176 @@
+
+You can also request a diff with each commit:
+
+```
+git log -p @metamask/<package-name>@<current-version>..HEAD -- packages/<package-name>
+```
+
+Or, you can view all of the changes in one:
+
+```
+git diff @metamask/<package-name>@<current-version>..HEAD -- packages/<package-name>
+```
+
+You'll reference these changes in upcoming steps.
+
+## 2. Determine impact of changes
+
+Now that you know what sort of changes are being made, the next step is look through the version bumps in the release PR and verify the impact of those changes so that it can be properly communicated.
+
+First, it's helpful to understand what sort of changes are worth communicating. While a release could certainly ship with a list of every single change made to the code in this repository, this would be needless. The people who will end up using the packages in this monorepo — consumers — only care about that part of code they can actually see and work with. So when it comes to understanding impact, this is what you should pay attention to:
+
+- Changes to interfaces
+- Changes to behavior
+- Changes to runtime dependencies
+- Changes to consumer-facing documentation
+
+Conversely, you generally should not need to worry about:
+
+- Changes to internal tooling
+- Changes to developer-only dependencies
+- Changes to contributor-only documentation
+
+With that in mind, there are four categories of changes that determine the level of impact.
+
+### Breaking changes
+
+If a consumer upgrades a package to a new version without making any modifications to their code, that version is "breaking" if it causes:
+
+- An error at runtime
+- An error at compile time (e.g., a TypeScript type error)
+- An error at install time (e.g., the consumer's package manager reports a Node version incompatibility)
+- A surprising difference in behavior
+
+Given this, there are many ways a change could be regarded as breaking:
+
+- Changing a function or method to throw an error
+- Adding a required argument to a function or method
+- Adding a property to a TypeScript type
+- Narrowing the type of a property in a TypeScript type
+- Narrowing the type of an argument in a function
+- Adding a parameter to a TypeScript type
+- Removing a parameter from a TypeScript type
+- Upgrading a runtime dependency to a version which causes any of the above
+- Removing an export from the package
+- Renaming the package
+- Bumping the Node version
+- Bumping the NPM version
+
+### Additions
+
+Minimally, this category could cover any actions that extend the surface area of a package in some way:
+
+- Adding an optional property to an existing type
+- Adding an optional argument to an existing function or method
+- Adding a new export to a package
+
+However, it is important to remember that consumers care about capabilities first and interfaces second. So, some additions could also be more simply regarded as changes. For instance:
```

Comment:
I'm not entirely sure what "some additions could also be more simply regarded as changes" means here. The example given, adding a new dependency, doesn't suggest any "addition" to the package itself. Is that what you meant to communicate - that not every "addition" adds to the package?

Reply:
Yes. I'm trying to clarify that even if you are technically adding something new that wasn't there before, it doesn't mean that the change needs to be categorized as an addition. 

That said, it's confusing that this is here. I will figure out another place to put this.

Reply:
I updated the section on categories to fit the intersection between SemVer and Keep a Changelog and removed this confusing wording.